docs/limitations.md

```diff
@@ -98,4 +99,17 @@ In the past, we used Ubuntu 18.04 as a base image and used older xfsprogs whenev
 TopoLVM assumes that PersistentVolumes created via Snapshotting or Cloning have the same storage class as the original PersistentVolume. However, this assumption is not verified on PersistentVolume creation with [external-provisioner in version `v3.2` or higher](https://github.com/kubernetes-csi/external-provisioner/blob/v3.2.0/CHANGELOG/CHANGELOG-3.2.md#feature).
 This was originally introduced to support changes for cloud-providers where storage-class attributes might change ([see the PR for implementation details](https://github.com/kubernetes-csi/external-provisioner/pull/699)) during the restore process, however this doesn't apply for TopoLVM.
 
-Thus, if a pod consumes a restored/cloned PV having a different storage class from the original PV, this pod will not get scheduled if the StorageClass contents differ from the source StorageClass (e.g. by using a different device class).
\ No newline at end of file
+Thus, if a pod consumes a restored/cloned PV having a different storage class from the original PV, this pod will not get scheduled if the StorageClass contents differ from the source StorageClass (e.g. by using a different device class).
+
+## LVMD seems to be leaking memory
+
+The memory usage of LVMD continues to increase, and it may appear that a memory leak is occurring. According to our investigations, we haven't found any heap memory leaks, it seems that the page cache is growing. 
+
+You can check this using the following metric expression. As you can see, the growth is offset by subtracting the cache size from the total usage.
+
+```
+container_memory_usage_bytes{namespace="topolvm-system", container="lvmd"} - container_memory_cache{namespace="topolvm-system", container="lvmd"}
+```
+
+We are dealing with this issue by reporting this issue to LVM mailing list.
+https://lore.kernel.org/linux-lvm/ZjO_tW3_5ZNhnHFm@redhat.com/T/
```

Comment:
The link here points to quite am outdated / possibly stale discussion. Is there a plan to revive it?

Reply:
I asked my colleague about it. He said that he was currently unable to work on it due to other work, but that he would like to resolve it.